src/main/java/uk/ac/ebi/ena/readtools/loader/common/eater/DataConsumer.java

```diff
@@ -10,11 +10,9 @@
 */
 package uk.ac.ebi.ena.readtools.loader.common.eater;
 
-public interface 
-DataEater<T1, T2>
-{
-    void    cascadeErrors() throws DataEaterException;
-    void    eat( T1 object ) throws DataEaterException;
-    void    setEater( DataEater<T2, ?> dataEater );
+public interface DataConsumer<T1, T2> {

```

Comment:
The template parameters T1 and T2 need to be restricted to some actual allowed types. Let's at least create a marker interfaces so that we can better understand what can go in here.